isis/tests/FunctionalTestsChan1m32isis.cpp

```diff
@@ -0,0 +1,558 @@
+#include <QTemporaryDir>
+
+#include "chan1m32isis.h"
+#include "Fixtures.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "TestUtilities.h"
+#include "Histogram.h"
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+using namespace Isis;
+using ::testing::HasSubstr;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/chan1m32isis.xml").expanded();
+
+TEST(Chan1m32Isis, Chan1m32IsisTestFowardAscending) {
+  QTemporaryDir prefix;
+  QString cubeFileName = prefix.path() + "/chan1m32isisTEMP.cub";
+  QVector<QString> args = {"from=data/chan1m32isis/forwardAscending/M3T20090630T083407_V03_L1B_cropped.LBL",
+                           "loc=data/chan1m32isis/forwardAscending/M3T20090630T083407_V03_LOC_cropped.IMG",
+                           "obs=data/chan1m32isis/forwardAscending/M3T20090630T083407_V03_OBS_cropped.IMG",
```

Comment:
Could we copy these into the temporary testing directory that gets created and use the copied files here. If we don't there are a couple cubes that get created in the test data directory. Maybe using something like https://en.cppreference.com/w/cpp/filesystem/copy